src/components/GisidaLite/helpers.tsx

```diff
@@ -90,3 +94,34 @@ export const gsLiteStyle: Style | string = DIGITAL_GLOBE_CONNECT_ID
       version: 8,
     }
   : 'mapbox://styles/mapbox/satellite-v9';
+
+/**
+ * gets map zoom, center and bounds form structures or jurisdictions
+ * @param {StructureFeatureCollection | null} structures - structure points
+ * @param {Jurisdiction | null} jurisdiction - jurisdiction
+ */
+export const getZoomCenterAndBounds = (
+  structures: StructureFeatureCollection | null,
+  jurisdiction: Jurisdiction | null
+) => {
+  let mapCenter;
+  let mapBounds;
+  let zoom;
+  if (structures?.features?.length) {
+    mapBounds = GeojsonExtent(structures);
+  }
+  if (!mapBounds && jurisdiction?.geojson) {
+    mapBounds = GeojsonExtent(jurisdiction.geojson);
+  }
+  // get map zoom and center values
+  if (mapBounds) {
+    const centerAndZoom = viewport(mapBounds, [600, 400]);
```

Comment:
could we make these magic numbers params in the function? Thinking of this from an [open/closed](https://stackify.com/solid-design-open-closed-principle/) perspective.